renovate.json

```diff
@@ -49,5 +49,16 @@
       "^release/**",
       "^kustomize/base/**"
     ]
-  }
+  },
+  "regexManagers": [
+    {
+      "fileMatch": [
+        "(^|/|\\.)Dockerfile$",
```

Comment:
**Thought:**
I wonder why matching Docker files requires 2 regexes in [Renovate's examples](https://docs.renovatebot.com/presets-regexManagers/#regexmanagersdockerfileversions).
```suggestion
```
I think just one regex should suffice for us: `(^|/)Dockerfile[^/]*$`.
We only have two files to match: `Dockerfile` and `Dockerfile.debug`.
![Screenshot 2022-11-29 at 10 13 39 AM](https://user-images.githubusercontent.com/10292865/204567392-e1511d31-bccd-4ca1-867d-a5a7d7b296c8.png)